# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/indexedDB/ActorsParent.cpp
# Commit: 1e1e3972a3bd
# Full Hash: 1e1e3972a3bd19aa3591fcb7a114cf3bfc4367bc
# Author: Simon Giesecke <sgiesecke@mozilla.com>
# Date: 2020-04-01 21:26:59
# Regressor Bug: 1623278
# File Overlap Count: 1
# Description:
#   Bug 1623278 - Workaround to make gcc 8.3.0 happy. r=jstutte
#   
#   Depends on D65547
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D69170
# ==============================================================================

diff -r 2bcd8d19b32c -r 1e1e3972a3bd dom/indexedDB/ActorsParent.cpp
--- a/dom/indexedDB/ActorsParent.cpp	Wed Apr 01 09:54:24 2020 +0000
+++ b/dom/indexedDB/ActorsParent.cpp	Wed Apr 01 09:56:58 2020 +0000
@@ -17325,7 +17325,7 @@
     // If there is an unexpected directory in the idb directory, trying to
     // delete at first instead of breaking the whole initialization.
     if (NS_WARN_IF(!databaseFilenames.GetEntry(subdirNameBase)) &&
-        NS_WARN_IF(NS_FAILED(DeleteFilesNoQuota(directory, subdirName)))) {
+        NS_WARN_IF((NS_FAILED(DeleteFilesNoQuota(directory, subdirName))))) {
       REPORT_TELEMETRY_INIT_ERR(kQuotaExternalError, IDB_GetEntry);
       return NS_ERROR_UNEXPECTED;
     }
